__tests__/api.test.js

```diff
@@ -89,10 +89,26 @@ describe("/api/articles", () => {
           created_at: expect.any(String),
           author: expect.any(String),
           body: expect.any(String),
-          article_id: expect.any(Number),
+          article_id: 1,
         })
       );
       expect(body.comments).toBeSortedBy("created_at", { descending: true });
     });
+    it("GET: 404 sends an error if the given article_id does not have any related comments", async () => {
+      const { body } = await request(app)
+        .get("/api/articles/2/comments")
+        .expect(404);
+      expect(body.message).toBe("There are no comments for article by id 2");
```

Comment:
Well done for implementing a different behaviour when there are no comments associated with an article, that said the ideal response in this case would be 200 with an empty array. It will be clearer why when we get into frontend 🙂